pkg/backend/backend.go

```diff
@@ -212,7 +212,7 @@ type Backend interface {
 	// LogoutAll logs you out of all the backend and removes any stored credentials.
 	LogoutAll() error
 	// Returns the identity of the current user and any organizations they are in for the backend.
-	CurrentUser() (string, []string, error)
+	CurrentUser() (string, []string, *workspace.TokenInformation, error)
```

Comment:
Can we change the return type to `Account, error` since it would simplify producing and using the result?

Reply:
We _could_ but we need to return this from filestate as well which makes `Account` look a little odd given the only field we can fill in is "Username".

Reply:
`CurrentUser()` already seems like a very httpstate backend specific method (filestate backend is returning the current OS user) and consolidating the return values would make the call-sites more consistent.

Reply:
This turned into a rabbit hole. Going to suggest sticking with it as in this PR to get his over the line, but to revisit when we've got some time to look into it again.

Reply:
SGTM